app/messaging/send-alert.js

```diff
@@ -0,0 +1,15 @@
+const { MessageSender } = require('ffc-messaging')
+const { messageConfig } = require('../config')
+const { createMessage } = require('./create-message')
+
+const sendAlert = async (body) => {
+  const message = createMessage(body)
+  const sender = new MessageSender(messageConfig.alertTopic)
+  await sender.sendMessage(message)
+  await sender.closeConnection()
+  console.log('Request for alert sent')
```

Comment:
I assume this is one of the console.log's we need to keep for our benefit?

Reply:
Yeah I thought it would be useful if we could see in the logs when an alert is sent.  It might be useful it might not.  But figured it's better to start with it and remove it if it's not useful later, than not have it and then wish it was there when we need it.

Reply:
Yep, I would agree